ng/src/web/pages/ldap/ldappage.js

```diff
@@ -0,0 +1,239 @@
+      return (
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableData>
+                {_('Activation')}
+              </TableData>
+              <TableData>
+                {certificate_info.activation_time}
+              </TableData>
+            </TableRow>
+            <TableRow>
+              <TableData>
+                {_('Expiration')}
+              </TableData>
+              <TableData>
+                {certificate_info.expiration_time}
+              </TableData>
+            </TableRow>
+            <TableRow>
+              <TableData>
+                {_('MD5 Fingerprint')}
+              </TableData>
+              <TableData>
+                {certificate_info.md5_fingerprint}
+              </TableData>
+            </TableRow>
+            <TableRow>
+              <TableData>
+                {_('Issued by')}
+              </TableData>
+              <TableData>
+                {certificate_info.issuer}
+              </TableData>
+            </TableRow>
+          </TableBody>
+        </Table>
+      )
+    }
+    return {};
+  }
+
+  render() {
+    const {loading} = this.state;
+    if (loading) {
+      return <Loading/>
+    }
+
+    const {
+      authdn,
+      certificate_info,
+      enable,
+      ldaphost,
+    } = this.state;
+
+    return (
+      <Layout flex="column">
+        <HelpIcon
+          page="users"
+          anchor="peruserldapauthentication"
+          title={_('Help: LDAP per-User Authentication')}
+        />
+        <Section
+          img="ldap.svg"
+          title={_('LDAP per-User Authentication')}
+        />
+
+        <Layout flex="column">
+          <FormGroup title={_('Enable')} titlesize="5">
+            <CheckBox
+              name="enable"
+              checked={enable === YES_VALUE}
+              checkedValue={YES_VALUE}
+              unCheckedValue={NO_VALUE}
+              onChange={this.handleValueChange}
+            />
+          </FormGroup>
+          <FormGroup title={_('LDAP Host')} titlesize="5">
+            <TextField
+              name="ldaphost"
+              value={ldaphost}
+              size="30"
+              onChange={this.handleValueChange}
+            />
+          </FormGroup>
+          <FormGroup title={_('Auth. DN')} titlesize="5">
+            <TextField
+              name="authdn"
+              value={authdn}
+              size="30"
+              onChange={this.handleValueChange}
+            />
+          </FormGroup>
+          <FormGroup title={_('CA Certificate')} titlesize="5">
+            <Layout flex="column">
+              {this.showCertfificateInfo(certificate_info)}
+              <FileField
+                name="certificate"
+                onChange={this.handleValueChange}
+              />
+            </Layout>
+          </FormGroup>
+          <Layout align="center">
+            <Button
+              width="auto"
+              onClick={() => this.handleSaveSettings()}
```

Comment:
Not required to use a arrow function here anymore. this.handleSaveSettings is already bound.